mne_connectivity/spectral/epochs.py

```diff
@@ -814,6 +815,20 @@ def spectral_connectivity_epochs(
             C = ----------------------
                 sqrt(E[Sxx] * E[Syy])
 
+        'cacoh' : Canonical Coherence (CaCoh) :footcite:`VidaurreEtAl2019`
```

Comment:
Was this reference added to the `references.bib`? https://github.com/mne-tools/mne-connectivity/blob/main/doc/references.bib

Reply:
Yes, this was already included from a mention in the MIC & MIM example:

https://github.com/mne-tools/mne-connectivity/blob/8393f5a015ada3c228f01deb37d8f28adec60b55/examples/mic_mim.py#L430-L434

https://github.com/mne-tools/mne-connectivity/blob/8393f5a015ada3c228f01deb37d8f28adec60b55/doc/references.bib#L229-L237